### PR TITLE
EJ2-Documentation-853041-Ug-Documentation-For-Row-Selection

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

**Review by VigneshNatarajan27 — CHANGES_REQUESTED** [review]
selectColumns API links is provided instead of selectRows API link


Select grid rows based on certain conditions topic, Javascript platform name is not properly displayed

**Comment by harinikrh** [comment]
> selectColumns API links is provided instead of selectRows API link
> 
> Select grid rows based on certain conditions topic, Javascript platform name is not properly displayed

Changed the API link for selectRows
Changed in to platform name

**Review by VigneshNatarajan27 — CHANGES_REQUESTED** [review]
persistSelection spelling is wrong

File: ej2-javascript/code-snippet/grid/grid-cs184/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    { field: 'CustomerID', headerText: 'Customer Name', width: 150 },
+    { field: 'OrderDate', headerText: 'Order Date', width: 130, format: 'yMd', textAlign: 'Right' },
+    { field: 'Freight',headerText:'Freight', width: 120, format: 'C2', textAlign: 'Right' },
+    { field: 'ShipCountry', headerText: 'Ship Country', width: 170 }
+  ],
+  height: 365
+});
+grid.appendTo('#Grid');
+
+var mod;
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/grid-cs184/index.js:17]: always I insist to provide meaningful name

File: ej2-javascript/code-snippet/grid/grid-cs184/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  if (page === 1) {
+    if (grid.pagerModule.pagerObj.currentPage != 1) {
+      grid.pagerModule.goToPage(1);
+    }
+    grid.selectRow(mod);
+  }
+  else {
+    grid.pagerModule.goToPage(page);
+    if (grid.pagerModule.pagerObj.currentPage == page) {
+      grid.selectRow.mod;
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/grid-cs184/index.js:63]: is this code is correct. grid.SelectRow.mod is not a proper code check on it

File: ej2-javascript/code-snippet/grid/grid-cs185/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    { field: 'OrderID', headerText: 'Order ID', textAlign: 'Right', width: 120 },
+    { field: 'CustomerID', headerText: 'Customer ID', width: 150 },
+    { field: 'ShipCity', headerText: 'Ship City', width: 150 },
+    { field: 'ShipName', headerText: 'Ship Name', width: 150 }
+  ],
+  height: 315,
+});
+grid.appendTo('#Grid');
+
+var enableSelection = false;
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/grid-cs185/index.js:14]: is this variable needed here

File: ej2-javascript/code-snippet/grid/grid-cs185/index.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    { field: 'OrderID', headerText: 'Order ID', textAlign: 'Right', width: 120 },
+    { field: 'CustomerID', headerText: 'Customer ID', width: 150, textAlign: 'Right' },
+    { field: 'ShipCity', headerText: 'Ship City', width: 150, textAlign: 'Right' },
+    { field: 'ShipName', headerText: 'Ship Name', width: 150, textAlign: 'Right' }
+  ],
+  height: 315
+});
+grid.appendTo('#Grid');
+
+let enableSelection = false;
```
Comment (MaithiliyK) [inline ej2-javascript/code-snippet/grid/grid-cs185/index.ts:20]: is the variable needed here

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

File: ej2-javascript/code-snippet/grid/grid-cs184/index.js (hunk 2 above)
Comment (harinikrh) [reply ej2-javascript/code-snippet/grid/grid-cs184/index.js:63]: changed in to grid.SelectRow(mod)

File: ej2-javascript/code-snippet/grid/grid-cs185/index.js (hunk 3 above)
Comment (harinikrh) [reply ej2-javascript/code-snippet/grid/grid-cs185/index.js:14]: Removed the variable

File: ej2-javascript/code-snippet/grid/grid-cs185/index.ts (hunk 4 above)
Comment (harinikrh) [reply ej2-javascript/code-snippet/grid/grid-cs185/index.ts]: Removed the variable

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

File: ej2-javascript/code-snippet/grid/grid-cs184/index.js (hunk 1 above)
Comment (harinikrh) [reply ej2-javascript/code-snippet/grid/grid-cs184/index.js]: changed the variable name

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Review by VigneshNatarajan27 — COMMENTED** [review]
changes are fine. since it is duplicate of already reviewed PR

**Pull request merged**